specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -16,119 +17,119 @@ enum TextCategory {
 }
 
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "MUST fix in next update"
-@doc("Image analyze category")
+@doc("Image analyze category.")
 enum ImageCategory {
   Hate,
   SelfHarm,
   Sexual,
   Violence,
 }
 
-@doc("The analysis request of the text.")
+@doc("The type of text analysis output.")
+enum AnalyzeTextOutputType {
```

Comment:
I'm slightly uncomfortable in calling this "outputType", as it affects only one aspect of the output -- the severity score. Would "AnalyzeTextSeverityType" be more descriptive?

Reply:
If there is other kind of score (for example float type score in [0,1], which is not severity levels), we will reuse this field and set proper value.
And since renaming this field would be a breaking change to our current status, I would keep it no change.